### PR TITLE
Prefixed persistence tables for Postgres for disambiguation between different networks

### PR DESCRIPTION
**Pull request by alexandrosfilios**

### CODE REVIEW

File: pkg/utils/errors/errors_test.go
Comment (adecaro) [inline pkg/utils/errors/errors_test.go:1]: please, could you increase unit-test coverage to 100%? It is at 70% now.

File: platform/common/core/generic/vault/txidstore/simpletxidstore.go [hunk 1]
```diff
@@ -116,49 +117,52 @@ func (s *SimpleTXIDStore[V]) Set(txID core.TxID, code V, message string) error {
 	// }
 
 	// 1: increment ctr in UnversionedPersistence
+	logger.Debugf("Incrementing ctr value in Unversioned persistence: %d", s.ctr)
```
Comment (adecaro) [inline platform/common/core/generic/vault/txidstore/simpletxidstore.go:120]: do we still this log?

File: platform/common/core/generic/vault/txidstore/simpletxidstore.go [hunk 2]
```diff
@@ -116,49 +117,52 @@ func (s *SimpleTXIDStore[V]) Set(txID core.TxID, code V, message string) error {
...
 	err := setCtr(s.Persistence, s.ctr+1)
 	if err != nil { // TODO: && !errors2.HasCause(err, UniqueKeyViolation)
 		s.Persistence.Discard()
-		return errors.Errorf("error storing updated counter for txid %s [%s]", txID, err.Error())
+		return errors.Wrapf(err, "error storing updated counter for txid %s", txID)
 	}
 
 	// 2: store by counter
+	logger.Debugf("Store TX [%s] with status [%v] by counter: %d", txID, code, s.ctr)
```
Comment (adecaro) [inline platform/common/core/generic/vault/txidstore/simpletxidstore.go:128]: do we still this log?

File: platform/common/core/generic/vault/txidstore/simpletxidstore.go [hunk 3]
```diff
@@ -116,49 +117,52 @@ func (s *SimpleTXIDStore[V]) Set(txID core.TxID, code V, message string) error {
...
 	}
 	err = s.Persistence.SetState(txidNamespace, keyByCtr(s.ctr), byCtrBytes)
 	if err != nil { // TODO: && !errors2.HasCause(err, UniqueKeyViolation)
 		s.Persistence.Discard()
-		return errors.Errorf("error storing ByNum for txid %s [%s]", txID, err.Error())
+		return errors.Wrapf(err, "error storing ByNum for txid %s", txID)
 	}
 
 	// 3: store by txid
+	logger.Debugf("Store TX [%s] with status [%v] by txid", txID, code)
```
Comment (adecaro) [inline platform/common/core/generic/vault/txidstore/simpletxidstore.go:145]: also this?

File: platform/fabric/core/generic/committer/committer.go [hunk 4]
```diff
@@ -81,6 +82,8 @@ type Service struct {
 	FabricFinality     FabricFinality
 	Tracer             tracing.Tracer
 
+	logger committer.Logger
```
Comment (adecaro) [inline platform/fabric/core/generic/committer/committer.go:85]: can't we have a logging package under common/core instead?

File: platform/view/services/db/driver/sql/common/base.go [hunk 5]
```diff
@@ -13,6 +13,7 @@ import (
 	"strings"
 	"sync"
 
+	errors2 "github.com/hyperledger-labs/fabric-smart-client/pkg/utils/errors"
```
Comment (adecaro) [inline platform/view/services/db/driver/sql/common/base.go:16]: if we import this, can we avoid to import "github.com/pkg/errors"?

**Review by adecaro — APPROVED** [review]
LGTM modulo comments. Please, apply before merging. Thanks :)

File: platform/view/services/db/driver/sql/common/base.go (hunk 5 above)
Comment (alexandrosfilios) [reply platform/view/services/db/driver/sql/common/base.go:16]: There are more methods I need from `pkg/errors` unfortunately

File: platform/fabric/core/generic/committer/committer.go (hunk 4 above)
Comment (alexandrosfilios) [reply platform/fabric/core/generic/committer/committer.go:85]: Sure let's align them all in a separate PR

**Pull request merged**